README.md

```diff
@@ -174,6 +174,66 @@ Once installed, you may use `mcsema-disass` for disassembling binaries, and `mcs
 
 In order to verify that McSema works correctly as built, head on over to [the documentation on integration tests](docs/AddingNewTests.md). Check that you can run the tests and that they pass.
 
+### On Windows
```

Comment:
I'm wondering if we should move installation/build instructions to their own per-OS documents, and just link to them from the README.md. Thoughts? (we can track this with a separate issue).